src/lib.rs

```diff
@@ -6,43 +6,53 @@ use std::{fs, process};
 
 use anyhow::{bail, ensure, Context, Result};
 use notify::{RecommendedWatcher, RecursiveMode, Watcher};
-use structopt::StructOpt;
+use structopt::{lazy_static, StructOpt};
 use wasm_bindgen_cli_support::Bindgen;
 
+pub fn get_metadata() -> &'static cargo_metadata::Metadata {
+    lazy_static::lazy_static! {
+        static ref METADATA: cargo_metadata::Metadata = cargo_metadata::MetadataCommand::new()
+            .exec()
+            .expect("cannot get crate's metadata");
+    }
+
+    &METADATA
+}
+
 #[derive(Debug, StructOpt)]
 pub struct Build {
     #[structopt(long)]
     release: bool,
+    #[structopt(skip = default_build_command())]
+    command: process::Command,
+}
+
+fn default_build_command() -> process::Command {
+    let mut command = process::Command::new("cargo");
+
+    command.args(["build", "--target", "wasm32-unknown-unknown"]);
+
+    command
 }
 
 impl Build {
     pub fn execute(
-        &self,
+        self,
         crate_name: &'static str,
```

Comment:
No idea why this would require to be static xD
```suggestion
        crate_name: &str,
```